Source/WebKit/Shared/WebHitTestResultData.h

```diff
@@ -89,6 +90,7 @@ struct WebHitTestResultData {
     enum class ElementType : uint8_t { None, Audio, Video };
     ElementType elementType;
     std::optional<FrameInfoData> frameInfo;
+    std::optional<WebCore::RemoteUserInputEventData> remoteUserInputEventData;
```

Comment:
Is this actually being used right now? If so, don't you need to change `WebHitTestResultData.serialization.in`?